tracing-attributes/tests/async_fn.rs

```diff
@@ -243,6 +258,13 @@ fn async_fn_with_async_trait_and_fields_expressions_with_generic_parameter() {
                 .with_field(field::mock("Self").with_value(&std::any::type_name::<TestImpl>())),
         )
         .enter(span2.clone())
+        .new_span(
+            span4
+                .clone()
+                .with_field(field::mock("Self").with_value(&std::any::type_name::<TestImpl>())),
+        )
+        .enter(span4.clone())
+        .exit(span4.clone())
```

Comment:
looks like clippy complains about the `clone` here because this is the last use of `span4`
```suggestion
        .exit(span4)
```